modules/carto/src/api/common.ts

```diff
@@ -1,10 +1,13 @@
+import { VERSION } from "@deck.gl/core";
+
 export const DEFAULT_API_BASE_URL = 'https://gcp-us-east1.api.carto.com';
 export const DEFAULT_CLIENT = 'deck-gl-carto';
 export const V3_MINOR_VERSION = '3.4';
 export const MAX_GET_LENGTH = 8192;
 
 export const DEFAULT_PARAMETERS = {
-  v: V3_MINOR_VERSION
+  v: V3_MINOR_VERSION,
+  deckglVersion: VERSION
```

Comment:
This will add the version to all requests made, not just instantiation

Reply:
in what other request? I've done a quick local test and It is only added in the first request. Maybe I'm forgetting something